CONTRIBUTING.md

```diff
@@ -108,6 +108,27 @@ resulting ``_build`` directory it produces; ``index.html`` is the main page.
 **NOTE:** You can try to use the `make` commands on Windows if you have `make` installed and something like `Cygwin`, `MSYS2` or `MYSYS`, or by using Windows Subsystem for Linux (`WSL`).
 
 
+
+## Configuring Netlify
+
+The steps to configure Netlify to preview pull requests are the following:
+
+* Pass a command to build the docs. In our case that command is
+
+      ci/install.sh && make docs
+
+* Pass the root directory of the generated docs. In our case that is
+
+      doc/_build/html/
+
+*NOTES*:
+
+* The `runtime.txt` file in the root of the repo is needed by Netlify to declare the Python version required to build the docs.
+* By default Netlify adds a lot of checks to pull requests, besides the one that actually allows to preview their changes live.
```

Comment:
```suggestion
* By default, Netlify adds a lot of checks to pull requests, besides the one that actually builds the live preview.
```

Fixed some grammar issues and clarified the phrasing

Reply:
Are you sure none of these checks are useful for us?

Reply:
We can check later.